.github/workflows/tests.yml

```
@@ -419,6 +419,7 @@ jobs:
 
   static-site:
     needs: process-results
+    if: always() && github.ref == 'refs/heads/main' && github.repository == 'coiled/coiled-runtime'
```

should we add a comment and why do we do this? 
Thinking something like this, not sure if the syntax of the comment is correct here. 
```suggestion
    # run always even when dependant workflow failed. 
    if: always() && github.ref == 'refs/heads/main' && github.repository == 'coiled/coiled-runtime'
```

---

Sure, good idea.